src/dispatch/__main__.py

```diff
@@ -7,15 +7,16 @@
 
 from dispatch.__init__ import (
     ARTIFACT_MAPPING,
```

Comment:
```suggestion
from dispatch.dispatch_globals import (
    ARTIFACT_MAPPING,
```
Imports broken, please repoint to a correct path.
Same for PACKAGE_MAPPING and other files, where the globals are used.

Reply:
Done, sorry about that.